### PR TITLE
fix logs menu for shunt compensator

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorModificationInfos.java [hunk 1]
```diff
@@ -61,6 +61,6 @@ public AbstractModification toModification() {
 
     @Override
     public Reporter createSubReporter(ReporterModel reporter) {
-        return reporter.createSubReporter(getType().name(), "Modification of shunt compensator " + getEquipmentId());
+        return reporter.createSubReporter(getType().name(), "shunt compensator modification ${shuntCompensatorId}", "shuntCompensatorId", this.getEquipmentId());
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorModificationInfos.java:64]: Capitalize first letter

File: src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorCreationInfos.java [hunk 2]
```diff
@@ -75,6 +75,6 @@ public AbstractModification toModification() {
 
     @Override
     public Reporter createSubReporter(ReporterModel reporter) {
-        return reporter.createSubReporter(getType().name(), "Creation of shunt compensator " + getEquipmentId());
+        return reporter.createSubReporter(getType().name(), "shunt compensator creation ${shuntCompensatorId}", "shuntCompensatorId", this.getEquipmentId());
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorCreationInfos.java:78]: Capitalize first letter

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorModificationInfos.java (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorModificationInfos.java]: done

File: src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorCreationInfos.java (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/modification/server/dto/ShuntCompensatorCreationInfos.java]: done

**Pull request merged**